code/API_definitions/application-profiles.yaml

```
@@ -33,59 +59,59 @@ paths:
     post:
       security:
         - openId:
-            - connectivity_insights:policy:write
+            - application-profile:write
       tags:
         - Application Profiles
       summary: |
         create a profile which represents an application's networking demands.
       description: |
         Define network monitoring intents for optimal end user application
         experience.
-      operationId: createProfile
+      operationId: createApplicationProfile
       requestBody:
         description: List of user-defined network quality thresholds
         content:
-          "*/*":
+          application/json:
             schema:
-              $ref: "#/components/schemas/NetworkQualityThresholds"
+              $ref: "#/components/schemas/ApplicationProfileRequest"
         required: true
       responses:
-        "200":
-          description: OK
+        "201":
+          description: Created
+          headers:
+            x-correlator:
+              $ref: "#/components/headers/x-correlator"
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/ApplicationProfileId"
+                $ref: "#/components/schemas/ApplicationProfile"
         "400":
           $ref: "#/components/responses/Generic400"
         "401":
           $ref: "#/components/responses/Generic401"
         "403":
           $ref: "#/components/responses/Generic403"
-        "404":
-          $ref: "#/components/responses/Generic404"
-        "409":
-          $ref: "#/components/responses/Generic409"
         "500":
           $ref: "#/components/responses/Generic500"
         "503":
           $ref: "#/components/responses/Generic503"
 
   /application-profiles/{applicationProfileId}:
-    put:
+    patch:
       security:
         - openId:
-            - connectivity_insights:application_profile:write
+            - application-profile:write
```

As `api-name` is _application-profiles_

```suggestion
            - application-profiles:write
```